src/sgame/sg_cmds.cpp

```diff
@@ -3497,6 +3497,98 @@ void Cmd_Reload_f( gentity_t *ent )
 	}
 }
 
+/*
+=================
+Cmd_TeamStatus_f
+=================
+*/
+void Cmd_TeamStatus_f( gentity_t * ent )
+{
+	int builders = 0;
+	gentity_t *tmp;
+	struct buildable_status_t
+	{
+		int count;
+		int health;
+		bool spawned;
+	} structures[ BA_NUM_BUILDABLES ] = {};
+
+	if ( g_teamStatus.Get() <= 0 ) 
+	{
+		trap_SendServerCommand( ent - g_entities, "print \"teamstatus is disabled.\n\"" );
+		return;
+	}
+
+	if ( ent->client->pers.namelog->muted ) 
+	{
+		trap_SendServerCommand( ent - g_entities, "print \"You are muted and cannot use message commands.\n\"" );
+		return;
+	}
+
+	if ( level.team[ ent->client->pers.team ].lastTeamStatus
```

Comment:
`ent->client->pers.team` -> `G_Team( ent )` is much more readable (and future-proof).
Also, you could do `int& lastTeamStatus = level.team[ G_Team( ent ) ].lastTeamStatus;` to make things much more readable.
For example,
`&& ( level.time - level.team[ ent->client->pers.team ].lastTeamStatus ) < g_teamStatus.Get() * 1000 )`
would become `&& ( level.time - lastTeamStatus ) < g_teamStatus.Get() * 1000 )` which could be compacted with previous line to give this:
`if ( lastTeamStatus && ( level.time - lastTeamStatus ) < g_teamStatus.Get() * 1000 )`

As you can see, the whole `if` expression can then stand on a single line and be shorter than the 2nd line of current one. 
￼